app/lib/utils.dart

```diff
@@ -36,10 +37,14 @@ Future<List<String>> listTarball(String path) async {
 }
 
 Future<String> readTarballFile(String path, String name) async {
-  final result = await Process.run('tar', ['-O', '-xzf', path, name]);
+  final result = await Process.run(
+    'tar',
+    ['-O', '-xzf', path, name],
+    stdoutEncoding: null,
```

Comment:
Now to be really annoying: should this be an optional argument? (allowing malformed)

I could see us being "tolerant" for the readme/changelog. It makes me more nervous for the pubspec.

Thoughts, @mkustermann ?